pytest/tests/sanity/rpc_state_changes.py

```diff
@@ -155,11 +158,12 @@ def test_changes_with_new_account_with_access_key():
     # Step 3
     status = nodes[0].get_status()
     latest_block_hash = status['sync_info']['latest_block_hash']
+    nonce += 8
```

Comment:
why 8?

Reply:
keep test exactly the same without nightly_protocol, it had `nonce=8` below.
`+=1` passes too